packages/server/src/provider/in-memory-provider/flag-configuration.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Don't export types from this file publicly.
+ * It might cause confusion since these types are not a part of the general API,
+ * but just for the in-memory provider.
```

Comment:
Don't we `export * from '@openfeature/shared` already in the SDK?

Reply:
I think what this comment means is to avoid exposing the `FlagConfiguration` type.

Reply:
Exactly

Reply:
Yeah, understood. I'm just trying to clarify if the comment is accurate.